dev-requirements.txt

```diff
@@ -21,8 +21,9 @@ async-timeout==4.0.2  # Used for timeouts in some test cases.
 # DOCUMENTATION #
 #################
 
-pdoc3==0.10.0
+pdoc==11.0.0
```

Comment:
Investigate code not being highlighted in all examples:

https://davfsa.github.io/hikari-docs/stable/hikari/permissions.html#Permissions